src/main/java/net/minecraftforge/oredict/OreNamePredicate.java

```diff
@@ -0,0 +1,43 @@
+package net.minecraftforge.oredict;
+
+import java.util.Iterator;
+import java.util.List;
+
+import net.minecraft.item.ItemStack;
+
+import com.google.common.base.Predicate;
+
+public class OreNamePredicate implements Predicate<ItemStack>, Iterable<ItemStack>
+{
+    public final String name;
+    private List<ItemStack> items; // lazily initiated to avoid load order issues
+
```

Comment:
There's a dependency loop causing NPE here: Vanilla items -> ```ToolMaterial``` -> this class -> ore dictionary -> Vanilla items